package com.valdosm.course.service.exceptions;

public class ResourceNotFoundException extends RuntimeException {
     public ResourceNotFoundException(Object id){
        super("Objeto n√£o encontrado. Id "+id);

     }
     

    }
    

